extensions/wikia/LanguageWikisIndex/styles/language-wikis-index.scss

```diff
@@ -0,0 +1,137 @@
+
+$language-wikis-index-header-background-color: #006CB0;
+$language-wikis-index-header-background-height: 105px;
+// Keep in sync with ThemeSettings::BACKGROUND_HEIGHT
+$language-wikis-index-header-background-height-small-breakpoint: 115px;
+$language-wikis-index-header-background-width: 430px;
+// Keep in sync with ThemeSettings::BACKGROUND_WIDTH
+$language-wikis-index-header-background-width-small-breakpoint: 471px;
+
+.language-wiki-index {
+	margin: 50px auto 0;
+	max-width: 710px;
+	text-align: center;
+
+	&__intro {
+		font-size: $wds-typescale-size-plus-2;
+		font-weight: $wds-fandom-font-weight-bold;
+		line-height: $wds-line-height;
+		margin: 20px auto;
+		max-width: 600px;
+	}
+
+	&__list-intro {
+		font-size: $wds-typescale-size-plus-1;
+		line-height: $wds-line-height;
+		margin: 20px;
+	}
+
+	&__list {
+		display: flex;
+		flex-wrap: wrap;
+		margin: 30px 0;
+	}
+
+	&__list-language {
+		margin-bottom: 10px;
+		width: 350px;
+
+		a {
+			color: $wds-fandom-color-link;
+		}
+	}
+
+	&__create-wiki {
+		background-color: $wds-fandom-color-light-gray;
+		display: flex;
+		flex-wrap: wrap;
+		margin: 20px 0;
+		padding: 15px 80px;
+	}
+
+	&__create-wiki-question {
+		margin: 0 auto;
+	}
+
+	&__create-wiki-link {
+		color: $wds-fandom-color-link;
+		font-weight: $wds-fandom-font-weight-bold;
+		margin: 0 auto;
+		text-transform: uppercase;
+	}
+
+	&__create-wiki-link-icon {
+		transform: rotate(180deg);
+		vertical-align: text-top;
+	}
+
+	&__links {
+		display: flex;
+		flex-wrap: wrap;
+		margin: 0 auto;
+		max-width: 600px;
+	}
+
+	&__links-item {
+		color: $wds-fandom-color-link;
+		width: 200px;
+	}
+
+	hr {
+		color: $wds-fandom-color-light-gray;
+	}
+}
+
+.language-wikis-index-header {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+	background-color: $language-wikis-index-header-background-color;
+	background-position: right center;
+	background-repeat: no-repeat;
+	background-size: contain;
+	height: $language-wikis-index-header-background-height;
+	margin: 0 auto;
+	position: relative;
+	z-index: 1;
+
+	@media #{$wds-breakpoint-l-down} {
+		height: $language-wikis-index-header-background-height-small-breakpoint;
+	}
+
+	@if ($widthType == 0) {
+		@media #{$breakpoint-small-only} {
+			width: $page-width-small + $padding-width-small * 2 + $content-border-width * 2;
+		}
+
+		@media #{$breakpoint-small-plus} {
+			width: $page-width-small-plus + $padding-width-small-plus * 2 + $content-border-width * 2;
+		}
+
+		@media #{$breakpoint-medium-only} {
+			width: $page-width-medium + $padding-width-medium * 2 + $content-border-width * 2;
+		}
+
+		@media #{$breakpoint-large-up} {
+			width: $page-width-large + $padding-width-large * 2 + $content-border-width * 2;
+		}
+	}
+
+	&::before {
+		background: linear-gradient(to right, rgba($language-wikis-index-header-background-color, .5), rgba($language-wikis-index-header-background-color, .5)),
```

Comment:
This is to have a gradient overlay over the background. I don't think we need it anymore.

Reply:
Whole `&::before` can be removed.